### DISCUSSION TITLE
安装 出现Extension hidden: 195

### DISCUSSION THREAD

**Discussion by nanhongchuan** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#277] [Q&A]
![1690102801130](https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111/assets/52574782/e8f7e6d5-7607-483c-9029-4d5755a3d15a)
安装出现这样的问题，请问应该如何处理 Extension hidden: 195。求解答。
![微信截图_20230723170155](https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111/assets/52574782/7971a264-a1a1-4c60-8eb9-584c45beb563)

**Comment by pkuliyi2015** [comment]
报错是说指定的文件夹已经存在。你进extensions文件夹，删了本插件的文件夹，再装即可
